### PR TITLE
chore: remove bcolors

### PR DESCRIPTION
**Pull request by fpgmaas**
**PR Checklist**

- [x] A description of the changes is added to the description of this PR.
- [ ] If there is a related issue, make sure it is linked to this PR.
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added or modified a feature, documentation in `docs` is updated

**Description of changes**

With the introduction of the new `Color` class, we probably don't need `bcolors` anymore, and I think we should remove it. I'm not 100% sure what the `blue` function does (why does it not reset, but change it to light red at the end?) But I'll leave that for another PR.

### CODE REVIEW

File: chispa/bcolors.py [hunk 1]
```diff
@@ -1,43 +0,0 @@
...
-    # Style
-    Bold = "\033[1m"
-    Underline = "\033[4m"
-
-
-def blue(s: str) -> str:
-    return bcolors.LightBlue + str(s) + bcolors.LightRed
-
-
-def underline_text(input_text: str) -> str:
```
Comment (SemyonSinchenko) [inline chispa/bcolors.py:39]: The only problem is that this function is exposed in API docs for some reason... I wold better deprecate it for couple of releases.

**Comment by SemyonSinchenko** [comment]
@fpgmaas Hi! Is there any chance you can update the PR?

**Comment by fpgmaas** [comment]
> @fpgmaas Hi! Is there any chance you can update the PR?

Hi @SemyonSinchenko , sorry for the late reaction. I updated the PR!

**Review by SemyonSinchenko — APPROVED** [review]
LGTM, thanks!

**Pull request merged**